pages/en/about/index.md

```diff
@@ -24,6 +24,7 @@ const server = http.createServer((req, res) => {
 });
 
 server.listen(port, hostname, () => {
+  // eslint-disable-next-line no-console
```

Comment:
this is an unfortunate addition to the displayed sample code. 

<img width="1552" alt="image" src="https://github.com/nodejs/nodejs.org/assets/298435/cdf192fe-a3bb-4e9e-94ef-ee61c7ac3d90">

This doesn't appear to trip `lint`...

If we absolutely need it, can we instead add this to the top of the file?

Reply:
Oh, we should remove this, yes.

Reply:
resolved via https://github.com/nodejs/nodejs.org/pull/6028/commits/6ff252f663d20831f39710e789db4dd6e8e8a038